src/libs/migrations/PersonalDetailsByAccountID.js

```diff
@@ -251,12 +251,6 @@ export default function () {
                     delete newReport.lastActorEmail;
                 }
 
-                if (lodashHas(newReport, ['participants'])) {
-                    reportWasModified = true;
-                    Log.info(`[Migrate Onyx] PersonalDetailsByAccountID migration: removing participants from report ${newReport.reportID}`);
-                    delete newReport.participants;
-                }
```

Comment:
This is getting removed as we will be using it again soon and do not want the migration to unset the value. I am pretty sure this should be fine for now as nothing should be using or sending the old `participants` array.

Reply:
Ok cool that seems fine to me.

Reply:
> This is getting removed as we will be using it again soon and do not want the migration to unset the value.

Sorry, I didn't get. What we will be using again soon?

> I am pretty sure this should be fine for now as nothing should be using or sending the old participants array.

This looks good 👍

Reply:
the `report.participants` value - but with a different data structure.